pyi/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN ./build-prereqs.sh
 ENV LD_LIBRARY_PATH=/usr/local/lib
 RUN yum install -y libffi-devel libffi yajl
 ADD felix_requirements.txt felix_requirements.txt
-RUN pip install -U -r felix_requirements.txt
```

Comment:
If you split this into two `pip` commands and run the `python-etcd` one first then you should get the github version of python-etcd.

Should we pin this to a particular commit for repeatability?

Why do you add -e here?  That installs python-etcd as a development egg, which shouldn't be needed.